src/test/java/com/example/demo/ConsumeRestApplicationTests.java

```diff
@@ -2,12 +2,20 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.web.client.RestTemplate;
 
 @SpringBootTest
 class ConsumeRestApplicationTests {
 
 	@Test
 	void contextLoads() {
 	}
-
+	
+	@Test
+	public  void Restservice() {
+		
+			RestTemplate resttemplate = new RestTemplate();
+			Greeting greeting = resttemplate.getForObject("http://localhost:8080//greetingMood?mood=1", Greeting.class);
+			System.out.println(greeting.getName());
```

Comment:
This line will just print the output of the operation but the test will still succeed.
You want to replace this line with a proper assertion like `assertEquals("yourexpectation", greeting.getName())` so that the test will fail should the API behave in an unwanted way.

Run the tests by running a `mvn clean install` on your project to make sure that the build AND tests both succeed.